tests/main/xdg-settings/task.yaml

```diff
@@ -2,53 +2,40 @@ summary: Ensure snap userd allows settings via xdg-settings
 
 # Not supposed to work on Ubuntu Core systems as we don't have
 # a user session environment there
-systems: [-ubuntu-core-*]
+systems:
+    - -amazon-linux-2-*
```

Comment:
Maybe a comment here why all these systems are now excluded?

Reply:
Would you mind if I do a pass over all the tests that share this characteristic and unify the comments? We kind of have a patchwork of comments that try to say the same thing in different ways now.

Reply:
Either way is fine, I was mostly commenting because the old test just disabled ubuntu-core and the updated test disables a bunch more.

Reply:
Done now